File: Python/2062.py
n = int(input())
palavras = input().split()

for i in range(len(palavras)):
    if 'OB' in palavras[i] and len(palavras[i]) <=3:
        palavras[i] = 'OBI'
    if 'UR' in palavras[i] and len(palavras[i]) <=3:
        palavras[i] = 'URI'
        
nova = ' '.join(palavras)
print(nova)

File: Java/2062.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        sc.nextLine();
        String[] pal = sc.nextLine().split(" ");

        for(int i = 0; i < pal.length; i++){
            if(pal[i].contains("OB") && pal[i].length() <= 3){
                pal[i] = "OBI";
            }if(pal[i].contains("UR") && pal[i].length() <= 3){
                pal[i] = "URI";
            }
        }

        for(int i = 0; i < pal.length; i++){
            if(i == pal.length-1){
                System.out.println(pal[i]);
            }else{
                System.out.printf("%s ", pal[i]);
            }
        }
    }
}
